app/models/saml_user.rb

```diff
@@ -0,0 +1,9 @@
+class SamlUser < User
+  attr_accessor Devise.saml_session_index_key.to_sym
```

Comment:
Nice. So this was issue for not maintaining index for saml user, when it was in User model?